### PR TITLE
Fixes broken drone hatches on doors (finally)

### LINKED ISSUE #5590: Maintenance drone hatches misplaced
Maintenance drone hatch sprites are all bungled up, they pop up on the right side of the airlock, and in the middle of multitile airlocks.

Issue seem to be on all horizontal mutlitile airlocks and on newly constructed single tile airlocks.

### PR DESCRIPTION
**Pull request by alsoandanswer**
title

dreamseeker_tT37AGXigk

Fixes #5590

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!bugfix

File: code/game/machinery/doors/multi_tile.dm [hunk 1]
```diff
@@ -32,28 +17,32 @@
...
-	if (overlays != null)
+/obj/machinery/door/airlock/multi_tile/setup_hatch()
+	if(overlays != null)
+		hatch_image = null
 		hatch_image = image('icons/obj/doors/hatches.dmi', src, hatchstyle, closed_layer+0.1)
 		hatch_image.color = hatch_colour
+		// reset any rotation and transformation applied
 		switch(dir)
 			if(EAST)
+				to_world("east")
```
Comment (fernerr) [inline code/game/machinery/doors/multi_tile.dm:29]:
```suggestion
```

File: code/game/machinery/doors/multi_tile.dm [hunk 2]
```diff
@@ -32,28 +17,32 @@
...
+				hatch_image.transform = turn(hatch_image.transform, 90)
 				hatch_image.pixel_x = hatch_offset_x
 				hatch_image.pixel_y = hatch_offset_y
 			if(WEST)
-				hatch_image.transform = turn(hatch_image.transform, 180)
+				to_world("west")
```
Comment (fernerr) [inline code/game/machinery/doors/multi_tile.dm:34]:
```suggestion
```

File: code/game/machinery/doors/multi_tile.dm [hunk 3]
```diff
@@ -32,28 +17,32 @@
...
+				hatch_image.transform = turn(hatch_image.transform, 90)
 				hatch_image.pixel_x = hatch_offset_x
 				hatch_image.pixel_y = hatch_offset_y
 			if(NORTH)
-				hatch_image.transform = turn(hatch_image.transform, -90)
+				to_world("north")
```
Comment (fernerr) [inline code/game/machinery/doors/multi_tile.dm:39]:
```suggestion
```

File: code/game/machinery/doors/multi_tile.dm [hunk 4]
```diff
@@ -32,28 +17,32 @@
...
+				hatch_image.transform = turn(hatch_image.transform, 90)
 				hatch_image.pixel_x = hatch_offset_y
 				hatch_image.pixel_y = hatch_offset_x
 			if(SOUTH)
+				to_world("south")
```
Comment (fernerr) [inline code/game/machinery/doors/multi_tile.dm:44]:
```suggestion
```

**Review by fernerr — APPROVED** [review]

**Pull request merged**